go.mod

```diff
@@ -1,7 +1,12 @@
 module github.com/nats-io/nats.go
 
+go 1.14
```

Comment:
intentional? think we also need `go mod tidy` here

Reply:
Agree will fix.